Runtime/Scripts/MeshSyncServer.cs

```diff
@@ -506,6 +506,9 @@ public void StartServer()
     [SerializeField] private int  m_serverPort      = MeshSyncConstants.DEFAULT_SERVER_PORT;
 #if UNITY_EDITOR
     [SerializeField] bool m_foldServerSettings = true;
+
+    internal IDCCLauncher m_DCCInterop;
+    public UnityEngine.Object m_DCCAsset;
```

Comment:
I agree. Additionally, unless required, this should not be shown in the inspector as well

Reply:
I've made it internal now. It's shown in the inspector so that users can set assets on their MeshSyncServer to open it in the DCC tool more easily from inside Unity.